tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -3240,38 +3237,19 @@ describe('runs_table', () => {
           id: 'book1',
           color: '#000',
           run: "The Philosopher's Stone",
+          experimentName: 'book',
           selected: true,
         },
         {
           id: 'book2',
           color: '#111',
           run: 'The Chamber Of Secrets',
+          experimentName: 'book',
           selected: false,
         },
       ]);
     });
 
-    it('passes selected value of false if run is not in selectionMap', () => {
```

Comment:
This is no longer necessary, as the behavior is tested thoroughly in the underlying selector.